docs/deploy.md

```diff
@@ -0,0 +1,47 @@
+# Deploy Stargz Snapshotter
```

Comment:
The file name should be `INSTALL.md` , and should be linked from README.md: https://github.com/containerd/stargz-snapshotter#quick-start-with-kubernetes

Reply:
Put kubernetes section into INSTALL.md too, add link in README.md